Create ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md (#1753)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md

---
date: ""

> title: Refresh button includes smart rules list in microservice status page

Reviewer: ```suggestion
title: Refresh button on microservice status tab includes smart rules list
```

> title: Updated user interface refresh button to exclude SmartRules microservice status

Reviewer: ```suggestion
title: Refresh button no longer includes smart rules microservice status
```

Reviewer: Please change the file name accordingly

Reviewer 2: Done.

> title: Updated user interface refresh button to exclude SmartRules microservice status

Reviewer 2: ```suggestion
title: Refresh button includes smart rules list in microservice status page
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-55911
version: 1019.6.9
---

> The **Refresh** button on the microservice status details page includes the smart rules list.

Reviewer: ```suggestion
The **Refresh** button on the **Status** tab of the microservice details now includes the smart rules list.
```

> The user interface refresh button on the microservice detail view was updated to exclude the SmartRules microservice status.

Reviewer: ```suggestion
The **Refresh** button on the microservice details page does no longer include the smart rules microservice status.
```

Reviewer 2: Done.

> The user interface refresh button on the microservice detail view was updated to exclude the SmartRules microservice status.

Reviewer 2: ```suggestion
The **Refresh** button on the microservice status details page includes the smart rules list.
```

---
date: ""

> title: ui refresh button on microservice status exclude smartrules (#5522) [GRAFT][release/cd] (#5630)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Updated User Interface Refresh Button to Exclude SmartRules Microservice Status
```

Co-authored-by: Pawel Michalowski <pmic@softwareag.com>
product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-55911

> version: 1019.6.9

Author: The Claude AI suggested to use the following body:
```suggestion
The user interface refresh button was updated to exclude the SmartRules microservice status. This improves page load performance by not making an unnecessary call to retrieve the SmartRules status when refreshing the page. 

This change was contributed by Pawel Michalowski.
```
